edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/BackupAndRestoreTest.cs

```diff
@@ -169,6 +169,10 @@ async Task BackupAndRestoreCorruptBackupDataTest(ITransportSettings[] transportS
                 // Get new fixture to re-initialize the edge hub container.
                 protocolHeadFixture = EdgeHubFixtureCollection.GetFixture();
 
+                // Reconnect clients due to C# SDK bug where it illegally attempts to send through closed amqp link
```

Comment:
I don't have much context about this test, but I know that a new version of c# client sdk (1.21.3) just released today.  Please wait for a while to use new sdk version.